content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started with Entity Framework Core
```

Comment:
Is this what you meant:

## Topics Covered

This tutorial will cover the following topics:
- Introduction to Entity Framework Core
- Adding Entity Framework Core to a Project